src/main/java/io/jcoder/tutorials/ch12/CLI.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2020 - JCoder Ltd
+ */
+package io.jcoder.tutorials.ch12;
+
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+/**
+ * Command Line Interface Class used in the Vending Machine Project.
+ * 
+ * <p>
+ * https://jcoder.io/content/course/java/beginners/ch12/
+ * </p>
+ * 
+ * @author Andres Ricardo Perez
+ */
+public class CLI {
+
+	private VendingMachine machine;
```

Comment:
please use the same formatting style as in other chapters of this project, e.g. https://github.com/jcoderltd/tutorial-java-basics/blob/master/src/main/java/io/jcoder/tutorials/ch07/hashmap/JavaBasicHashMapApp.java (4 spaces)

Reply:
this applies to other classes as well :)